website/static/custom.css

```diff
@@ -68,9 +68,27 @@
     max-width: 200px;
 }
 
+ol {
+  /*border-top: 1px solid #D80029;
+  border-bottom: 1px solid #F82049;
+  font-style: italic;*/
+  background-color: rgba(216, 0, 41, 0.05);
+  border-left: 8px solid rgba(216, 0, 41, 0.2);
+  color: #555;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 10px;
+}
+
 blockquote {
   background-color: rgba(100, 188, 255, 0.3);
+  background-image: url(/docs/images/image6.png);
+  background-repeat: no-repeat;
+  background-size: 60px;
+  background-position: 5px 10px;
   border-left: 8px solid #64b0ff;
+  padding: 5px 70px;
```

Comment:
This is a fair bit of padding to have on mobile, if we're keeping the large **i** symbols perhaps we just have padding on the left, e.g.
````css
padding-left: 70px;
````

Reply:
Fraser suggested using an i symbol from fontawesome - presumably that could scale using a media rule. Or the i bitmap could shrink + reduce the padding. Personally I like the i to show that it's extra info/suggestion but that is just my opinion.